parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -165,24 +165,24 @@ coreos:
         ExecStart=/opt/azure/containers/provision-setup.sh
 {{else}}
 runcmd:
+- retrycmd_if_failure() { for i in $(seq 1 36); do $@; [ $? -eq 0  ] && break || sleep 5; done ; }
```

Comment:
Just curious, why we pick 36?

Reply:
5 * 36 = 180s = 3 minutes, this is a generous timeout to ensure custom script had time to run. I'll have a follow up PR to parametrize this and clean up the runcmd in general.